### PR TITLE
Add some documentation to the README.md

### PR DESCRIPTION
**Pull request by ian-r-rose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
-This is a serverextension and lab extension for running LaTeX from inside the jupyterlab interface.
+# JupyterLab Latex
```
Comment (willingc) [inline README.md]: Perhaps match the repo name "jupyterlab-latex" or use "JupyterLab LaTeX"

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+An extension for JupyterLab which allows for live-editing of LaTeX documents.
+
+To use, right-click on a `.tex` document within JupyterLab, and select `Show LaTeX Preview`.
```
Comment (willingc) [inline README.md]: Perhaps "Usage" header

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+This will compile the `.tex` file and open the rendered PDF document.
+Subsequent saves of the file will automatically update the PDF.
+If the PDF fails to compile (possibly due to a syntax error),
+an error panel will open detailing the LaTeX error.
+
+## Requirements
+* JupyterLab 0.29
```
Comment (willingc) [inline README.md]: Add space between header and bullet

File: README.md [hunk 4]
```diff
@@ -1 +1,29 @@
...
+* Something that can compile `.tex` files to PDF on your notebook server (e.g., `pdflatex` or `xelatex`).
```
Comment (willingc) [inline README.md]: Something -> an app or program

File: README.md [hunk 5]
```diff
@@ -1 +1,29 @@
...
+
+## Installation
+
+In order to enable this extension you must install both a notebook server extension and a lab extension.
```
Comment (willingc) [inline README.md]: I would be more specific here as to exactly what is the notebook server extension and what is the lab extension or are they both in this repo?

File: README.md [hunk 6]
```diff
@@ -1 +1,29 @@
...
+From the `jupyterlab-latex` directory, enter the following into your terminal:
+```bash
+pip install .
+jupyter serverextension enable jupyterlab_latex
```
Comment (willingc) [inline README.md]: Does order matter here? If not, I would do both installs and then the enable step.

File: README.md [hunk 7]
```diff
@@ -1 +1,29 @@
...
+jupyter labextension install .
+```
+The extension defaults to running `xelatex` on the server.
```
Comment (willingc) [inline README.md]: Perhaps add `### Customize your install` or `### Customization` header

File: README.md (hunk 5 above)
Comment (ian-r-rose) [reply README.md]: They are both in this repo. I have added some text clarifying.

File: README.md (hunk 6 above)
Comment (ian-r-rose) [reply README.md]: Nope, the order does not matter. I have changed it around.

**Comment by ian-r-rose** [comment]
Thanks, @willingc! I have addressed your comments.

**Comment by mpacer** [comment]
This is great!

**Pull request merged**